package.json

```diff
@@ -70,10 +70,10 @@
     "eslint": "5.13.0",
     "glob": "7.1.3",
     "husky": "1.3.1",
-    "jsdoc-to-markdown": "4.0.1",
+    "jsdoc-to-markdown": "^5.0.0",
```

Comment:
Breaking change is dropping Node 6 support, but Dynamoose is at Node 8 already anyway.